triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -60,10 +60,15 @@ object TriggerServiceFixture {
       mat: Materializer,
       aesf: ExecutionSequencerFactory,
       ec: ExecutionContext): Future[A] = {
-    // Launch a toxiproxy instance . Wait on it to be ready to accept
+    // Launch a toxiproxy instance. Wait on it to be ready to accept
     // connections.
     val host = InetAddress.getLoopbackAddress()
-    val toxiProxyExe = BazelRunfiles.rlocation("external/toxiproxy_nix/bin/toxiproxy-cmd")
+    val toxiProxyExe =
+      if (sys.props("os.name").toLowerCase.contains("windows")) {
```

Comment:
Please don’t do this. Define it as a `dev_env_tool` like I suggested and locate that.

Reply:
Gimme a break Moritz! I'm working on it!!

Reply:
Sorry I got a bit irritated that you just pushed this commit after my original suggestion to use `dev_env_tool` :slightly_smiling_face: